### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [susingha/NYTSearch#1]
Hello Supratik,

Looks like your submission missing the required feature: [ ] User can click on "settings" which allows selection of advanced search options to filter results. Can you please double-check that you completed the feature, update your code and/or README and then [re-submit your project](https://courses.codepath.com/courses/intro_to_android/pages/submitting_assignments#heading-submitting-through-course-system) through the course system.

**Comment by susingha** [comment]
Yup, missed for lack of time. Am on it now. Will re-submit in a couple of hours

**Comment by susingha** [comment]
@codepathreview I have updated the github code with all required User Stories completed. - Thanks

**Comment by codepathreview** [comment]
Satisfactory work. A few notes after checking out the code:
- Nice to see you properly reused the single base method in search activity to fetch results for both initial load and the pagination.
- Consider moving the search field into the ActionBar using a [custom ActionBar layout](http://guides.codepath.com/android/Extended-ActionBar-Guide#custom-actionbar-layout) or with a [SearchView](http://guides.codepath.com/android/Extended-ActionBar-Guide#adding-searchview-to-actionbar)
- Consider using DialogFragment. Fragment is an important component and its use is good practice.
- Consider using a separate class for API client.
- Consider using the folder structure and naming conventions defined in http://guides.codepath.com/android/Organizing-your-Source-Files

Also, here's a more detailed [Project 2 Feedback Guide here](http://courses.codepath.com/snippets/intro_to_android/project_2_feedback) which covers the most common issues with this submitted project. Read through the [feedback guide point-by-point](http://courses.codepath.com/snippets/intro_to_android/project_2_feedback) to determine how you could improve your submission.

Let us know if you have any other thoughts or questions about this assignment. The next assignment (Twitter Client) will be especially important since it introduces the majority of the remaining pieces necessary to build a fully functional API client with complex feeds of data and user creation.
